src/Microsoft.OpenApi.OData.Reader/Operation/EdmActionOperationHandler.cs

```diff
@@ -64,16 +64,6 @@ protected override void SetRequestBody(OpenApiOperation operation)
         protected override void SetExtensions(OpenApiOperation operation)
         {
             operation.Extensions.Add(Constants.xMsDosOperationType, new OpenApiString("action"));
```

Comment:
shouldn't this method call the base on (from operation) ?

Reply:
Could you please explain this a bit more?

Reply:
EdmActionOperationHandler derives from EdmOperationOperationHandler. They both implement SetExtensions with different aspects (set different things). It feels like the implementation in EdmActionOperationHandler  should also call base.SetExtensions so all the information is present. Unless I'm missing why we wouldn't set the information from the base?

Reply:
We are calling `base.SetExtensions(operation)` on line 67. Is that what you mean?

Reply:
hum. sorry, I missed it somehow, ignore my confused self.